src/solver/application.cpp

```diff
@@ -55,12 +75,13 @@ void checkSimplexRangeHydroPricing(Antares::Data::SimplexOptimization optRange,
 }
 
 // CHECK incompatible de choix simultané des options « simplex range= daily » et «
-// unit-commitment = MILP ».
+// unit-commitment = Accurate » ou « unit-commitment = MILP ».
 void checkSimplexRangeUnitCommitmentMode(Antares::Data::SimplexOptimization optRange,
                                          Antares::Data::UnitCommitmentMode ucMode)
 {
     if (optRange == Antares::Data::SimplexOptimization::sorDay
-        && ucMode == Antares::Data::UnitCommitmentMode::ucMILP)
+        && (ucMode == Antares::Data::UnitCommitmentMode::ucAccurate
+            || ucMode == Antares::Data::UnitCommitmentMode::ucMILP))
```

Comment:
```suggestion
        && (ucMode == UnitCommitmentMode::ucAccurate
            || ucMode == UnitCommitmentMode::ucMILP))
```
Add `using namespace Antares::Data;` as the 1st instruction in the function.